### PR TITLE
Achievement messages

### PR DESCRIPTION
**Pull request by Tsearo**
Add more specific edit messages when making updates to achievement logic, badge, title/description and points.

Reasoning is to eliminate the ambiguity of if someone broke the logic or just did a rescore/badge update.

Example of old vs new
image

### CODE REVIEW

File: lib/database/achievement.php [hunk 1]
```diff
@@ -597,11 +600,28 @@ function UploadNewAchievement(
...
+                    }
+
+                    $editString = implode(', ', $stringArray);
+
                     addArticleComment(
                         "Server",
                         \RA\ArticleType::Achievement,
                         $idInOut,
-                        "$author edited this achievement.",
+                        "$author edited this achievements $editString.",
```
Comment (Jamiras) [inline lib/database/achievement.php:624]: possessive form should be "achievement's".

File: lib/database/achievement.php [hunk 2]
```diff
@@ -527,13 +527,16 @@ function UploadNewAchievement(
...
-        $query = "SELECT Flags, Points, Author FROM Achievements WHERE ID='$idInOut'";
+        $query = "SELECT Flags, MemAddr, Points, Title, Description, BadgeName, Author FROM Achievements WHERE ID='$idInOut'";
         $dbResult = s_mysql_query($query);
         if ($dbResult !== false && mysqli_num_rows($dbResult) == 1) {
             $data = mysqli_fetch_assoc($dbResult);
 
             $changingAchSet = ($data['Flags'] != $type);
             $changingPoints = ($data['Points'] != $points);
+            $changingBadge = ($data['BadgeName'] != $badge);
+            $changingWording = ($data['Title'] != $title | $data['Description'] != $desc);
```
Comment (Jamiras) [inline lib/database/achievement.php:538]: two bars (`||`) for logical or.

Comment (Tsearo) [reply lib/database/achievement.php]: Oof missed that, thanks

File: lib/database/achievement.php (hunk 1 above)
Comment (Tsearo) [reply lib/database/achievement.php]: Thanks, should be correct now

**Review by Jamiras — APPROVED** [review]

**Pull request merged**